### PR TITLE
Edited placeholder text for all copy code snippets

### PR DESCRIPTION
**Pull request by clintmoss**
- Edited placeholder text for visual examples and for when code is copied.

### CODE REVIEW

File: src/pages/templates/TwoColumn2.js [hunk 1]
```diff
@@ -244,50 +244,51 @@ const [templateImage, setTemplateImage] = useState("/images/templates/twocolumn2
...
+        <div class="image-box"> <img src="https://picsum.photos/id/1018/400/200" alt="Writing Tools" /> </div>
+        <div class="image-box"> <img src="https://picsum.photos/id/1047/300/400" alt="Academic Books" /> </div>
+        <div class="image-box"> <img src="https://picsum.photos/id/1059/500/300" alt="Study Space" /> </div>
+        <div class="image-box"> <img src="https://picsum.photos/id/119/500/300" alt="Writing Desk" /> </div>
+        <div class="image-box"> <img src="https://picsum.photos/id/143/800/200" alt="Research Materials" /> </div>
       </div>
     </div>
-    <p>Vivamus suscipit tortor eget felis porttitor volutpat. Vivamus magna justo, lacinia eget consectetur sed, convallis at tellus. Proin eget tortor risus. Donec rutrum congue leo eget malesuada.</p>
+    <p>Use the images and resources provided to support your writing journey and...</p>
     <br>
```
Comment (franreyn) [inline src/pages/templates/TwoColumn2.js:270]: Please investigate why we have two `<br>` here.

**Review by franreyn — CHANGES_REQUESTED** [review]
Please take a look at the youtube videos, the ones I checked were "video unavailable". We can use placeholder videos (from youtube).

File: src/pages/templates/OneColumn3.js [hunk 2]
```diff
@@ -203,137 +203,133 @@ export default function OneColumn3() {
...
         <h3>Supplementary Video</h3>
         <div class="card-content">
-          <h4>Il faut</h4>
-          <p>(Video length: 3:50 minutes)</p>
+          <h4>Effective Writing</h4>
+          <p>(Video length: 4:12 minutes)</p>
           <div class="media-container">
             <div class="media-object">
-              <iframe src="https://www.youtube.com/embed/rKV0Yl1Ir2g" title="YouTube video player?wmode=opaque" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowFullScreen="" width="560" height="315" frameBorder="0"></iframe>
+              <iframe src="https://www.youtube.com/embed/XYZ12345678" title="YouTube video player?wmode=opaque" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowFullScreen="" width="560" height="315" frameBorder="0"></iframe>
```
[the remark this answers is not in the document]
Comment (clintmoss) [reply src/pages/templates/OneColumn3.js:305]: The link changed when I changed all the placeholder text in chat GPT. I'll be sure to keep an eye out for this from now on. Thanks. 

I've gone ahead and changed the link back.

File: src/pages/templates/TwoColumn2.js (hunk 1 above)
Comment (clintmoss) [reply src/pages/templates/TwoColumn2.js:270]: Removed the other <br>

**Review by franreyn — APPROVED** [review]
Thank you

**Pull request merged**